MaterialLib/SolidModels/CreepBGRa.cpp

```diff
@@ -43,9 +43,16 @@ CreepBGRa<DisplacementDim>::integrateStress(
     KelvinVector sigma_try = sigma_prev + C * (eps - eps_prev);
     ResidualVectorType solution = sigma_try;
 
+    const double A = _a(t, x)[0];
+    const double n = _n(t, x)[0];
+    const double sigma0 = _sigma_f(t, x)[0];
+    const double Q = _q(t, x)[0];
+
+    const double coef = A * std::pow(1.5, 0.5 * (1 + n)) / std::pow(sigma0, n);
```

Comment:
style: to name the variable after its meaning would be better.

The same function is implemented below; a single function for this computation might be better.

Reply:
Introduced a local function for this coefficient computation, and changed the variable name with meaning as well.